helm-chart/renku/Chart.yaml

```diff
@@ -3,4 +3,4 @@ appVersion: '1.0'
 description: A Helm chart for the Renku platform
 name: renku
 icon: https://github.com/SwissDataScienceCenter/renku-sphinx-theme/raw/master/renku_sphinx_theme/static/favicon.png
-version: 0.13.0
+version: 0.14.1
```

Comment:
where is `0.14.0`?

Reply:
this gets set automatically when the chart gets published (by chartpress) so it's not strictly necessary to have the correct value here